src/lsp-server.ts

```diff
@@ -225,40 +224,13 @@ export class LspServer {
         const offset: number = getOffsetOfLineAndCharacter(contents, params.position.line + 1, params.position.character + 1);
         const symbol: string = codeSelect(contents, offset);
         const language: string = path.extname(fileName);
-        return new Promise<Location[]>(resolve => {
-            // limit the serach scope within same file extension
```

Comment:
IMO it's better to add TODO to explain why we don't consider finding the references, located in `.cpp`, of the symbols located in `.h`, or cases like that.

Reply:
Done.